CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed a few unit tests that were failing on Windows due to different EOL character and path separators ([#393](https://github.com/Open-EO/openeo-python-client/issues/393))
```

Comment:
a changelog entry is not that necessary here,  unit test fixes  are generally not that valuable for the end user